bench/util/volumeRunner.js

```diff
@@ -0,0 +1,173 @@
+'use strict'
+// run this AFTER /database/loadVolumeData.js has populated db successfully
+const DEBUG = false // prints request/response details
+const START_SERVER = true // true = start udaru
+
+// ensure variables same as /database/loadVolumeData.js
+const NUM_TEAMS = 500 // total number of teams
+const TEAM_START_ID = 7 // user start id offset
+const NUM_USERS_PER_TEAM = 100
+const NUM_POLICIES_PER_TEAM = 10
+const DURATION = 15 // how long to run tests for in seconds
+
+const autocannon = require('autocannon')
+
+// part of initial route, this changes for second run
+var partialRoute = '/authorization/access/'
+
+var Server = null
+if (START_SERVER) {
+  // start udaru server
+  Server = require('../../lib/server/index')
+  Server.start((err) => {
+    if (err) {
+      return console.error(`Failed to start server: ${err.message}`)
+    }
+
+    console.log('Server started on: ' + Server.info.uri.toLowerCase())
+
+    startBench()
+  })
+} else {
+  startBench()
+}
```

Comment:
I would not recommend to run both autocannon and the benchmark on the same process. Both things can easily saturate a process at 100% cpu. You might want to spin up a child process with the server.

Reply:
good call, just added for convenience and is configurable

Reply:
sorry thought you meant udaru server... ignore my comment

Reply:
I meant udaru server, sorry. I know it’s a convenience, but it generates bad results. Spin it up using a child process.

Reply:
Thanks Matteo, I forked a child process now, I think I captured all cases to start up/shut down correctly etc. I've set server config variable to true as it's definitely convenient especially if it doesn't affect speed